vaadin-spring-tests/test-spring-security-fusion/src/main/java/com/vaadin/flow/spring/fusionsecurity/SecurityConfig.java

```diff
@@ -55,11 +58,23 @@ protected void configure(HttpSecurity http) throws Exception {
         setLoginView(http, "/login");
 
         if (stateless) {
-            setStatelessAuthentication(http,
-                    new SecretKeySpec(Base64.getUrlDecoder().decode(
+            // setStatelessAuthentication(http,
```

Comment:
could remove this commented code?

Reply:
Temporarily commented for testing. Maybe I should rather put it back, we probably want to keep `setStatelessAuthentication` method tested.